modules/ROOT/pages/endpoint-sftp-send.adoc

```diff
@@ -27,27 +38,27 @@ Send application messages (JSON or XML) to your backend applications in inbound
 |Yes
 
 |Username
-|Username for the SFTP Server.
+|The username for the SFTP Server.
 |Yes
 
 |Password
-|Password for the SFTP Server.
+|The password for the SFTP Server.
 |Yes
 
 |Target Path
-a|Path to the SFTP server EDI files:
+a|The path to the SFTP server EDI files:
 
 * That are generated from outbound message flows
 * Need to be placed for your trading partners
 * Where your backend applications require you to place translated JSON and XML files generated from inbound message flows
```

Comment:
Does this conceptual info belong in this table?
Also note that they are not parallel.
Oops...but I was trying to stay away from the legacy text issues. However, the text that you altered is tied to this....

Reply:
Done